tests/integration/targets/eos_pyateos/tests/eapi/pyateos.yaml

```diff
@@ -0,0 +1,131 @@
+---
+- debug: msg="START eapi/eos_pyateos.yaml on connection={{ ansible_connection }}"
+
+- name: set NTP, SNMP config.
+  arista.eos.eos_config:
+    lines:
+      - ntp server 216.239.35.8
+      - ntp server 216.239.35.4
+      - snmp-server host 8.8.8.4 version 2c public
+      - snmp-server host 8.8.8.8 version 2c public
+    provider: '{{ eapi }}'
+  become: true
+  register: result
+- assert:
+    that: result.changed == true
+
+- name: set PREFIX-LIST config.
+  arista.eos.eos_config:
+    lines:
+      - ip prefix-list T666 seq 10 permit 8.8.8.8/32
+    provider: '{{ eapi }}'
+  register: result
+  become: true
+- assert:
+    that: result.changed == true
+
+- name: set VLAN.
+  arista.eos.eos_config:
+    lines:
+      - vlan 666
+    provider: '{{ eapi }}'
+  register: result  
+  become: true
+- assert:
+    that: result.changed == true
+
+- name: run BEFORE tests.
+  arista.eos.eos_pyateos:
+    before: true
+    group: 
+      - mgmt
+    test:
+      - prefix_list
+      - vlan
+      - arp
+    hostname: "{{ inventory_hostname }}"
+    provider: '{{ eapi }}'
+  register: result
+- assert:
+    that: (result.before_file_ids|lenght=5)
```

Comment:
```suggestion
    that: (result.before_file_ids|length == 5)
```

Reply:
Sure. I need to set-up the environment. Then I ll make sure to run the test locally first